CONTRIBUTING.md

```diff
@@ -337,6 +337,10 @@ The exercise should consist of, at minimum:
 * A test suite
 * A reference solution that passes the test (see [#reference-solution](#reference-solution))
 
+You will need to add the exercise to `"exercises"` section of the `config.json` file in the track.
+The order in which the exercises are listed there is the order in which they are fetched by default by `exercism fetch`.
+Typically, exercises are ordered by difficulty, unless there is a particular reason to do otherwise.
```

Comment:
Given this, it may not be safe for this document to make a blanket prescription on exercise ordering. I am considering three choices:

1. Delete this line; do not replace.
2. Delete this line; replace with "Please check with the track to which you are submitting an exercise for guidance on how to order the exercises".
3. Append the above "Please check"... sentence to this line.